botocore/handlers.py

```diff
@@ -781,11 +781,25 @@ def _replace_content(self, section):
         section.write(updated_content)
 
 
+class CommandAlias(object):
+    def __init__(self, actual_name):
+        """ Aliases a non-extant method to an existing method.
+
+        :param actual_name: The name of the method that actually exists on
+            the client.
+        """
+        self._actual = actual_name
+
+    def alias_command(self, client, **kwargs):
```

Comment:
Or we can just make this a ``__call__``? Using the command notation is a little confusing for me.